src/libraries/System.Globalization/tests/IcuAppLocal/IcuAppLocal.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.RemoteExecutor;
+using System.Diagnostics;
+using System.Reflection;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class IcuAppLocalTests
+    {
+
+        private static bool SupportIcuPackageDownload => PlatformDetection.Is64BitProcess &&
+                                                         PlatformDetection.IsNotOSX &&
```

Comment:
I see that that the `Microsoft.ICU.ICU4C.Runtime` package is available for x86 (https://www.nuget.org/packages/Microsoft.ICU.ICU4C.Runtime.win-x86). Is the `PlatformDetection.Is64BitProcess` condition correct?

Reply:
I think the condition should look like this:

```
(PlatformDetection.IsWindows && 
    !PlatformDetection.IsArmProcess) ||
(PlatformDetection.IsLinux &&
    (PlatformDetection.IsX64Process || PlatformDetection.IsArm64Process) &&
    !PlatformDetection.IsAlpine &&
    !PlatformDetection.IsLinuxBionic);
```

Reply:
Your suggestion should work. I did it originally for 64-bit only because ICU x86 was not available for all OSs. Anyway, I applied your suggestion with a little modification with adding `RemoteExecutor.IsSupported`. This is needed because this property not only checks the platforms but also supports environment variable to disallow running remote execution. 
Also, `PlatformDetection` had `IsLinux` as private property so I made it public.

Thanks for the feedback. Let me know if you have any other feedback.